starfish/intensity_table/intensity_table.py

```diff
@@ -233,6 +237,9 @@ def load(cls, filename: str) -> "IntensityTable":
             loaded.coords,
             loaded.dims
         )
+        # maintain log
+        if LOG in loaded.attrs:
+            intensity_table.attrs[LOG] = loaded.attrs[LOG]
```

Comment:
does this need to be deserialized?

Reply:
does it? maybe I should just always store then json encoded info in both the Imagestack and Intensitytable. Since it's just for reading do we care about maintaining the original objects?